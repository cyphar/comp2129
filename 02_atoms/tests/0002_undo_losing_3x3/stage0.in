START 3 3 3
PLACE 0 0
PLACE 0 1
PLACE 2 2
PLACE 0 0
PLACE 2 2
DISPLAY
STAT
UNDO
UNDO
PLACE 1 0
PLACE 1 1
DISPLAY
STAT
