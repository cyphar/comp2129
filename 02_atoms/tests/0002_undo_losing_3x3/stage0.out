Game Ready
Red's Turn

Green's Turn

Purple's Turn

Red's Turn

Purple's Turn

Red's Turn


+--------+
|  |R1|  |
|R2|  |P1|
|  |P1|  |
+--------+

Player Red:
Grid Count: 2

Player Green:
Lost

Player Purple:
Grid Count: 2

Purple's Turn

Red's Turn

Green's Turn

Purple's Turn


+--------+
|R1|R1|  |
|G1|G1|  |
|  |  |P1|
+--------+

Player Red:
Grid Count: 2

Player Green:
Grid Count: 2

Player Purple:
Grid Count: 1

